src/content/reference/react/useMemo.md

```diff
@@ -629,26 +629,25 @@ export default function TodoList({ todos, tab, theme }) {
 }
 ```
 
-The behavior would be the same. If the `visibleTodos` haven't changed, `List` won't be re-rendered.
+Perilakunya akan sama. Jika `visibleTodos` tidak berubah, `List` tidak akan di*render* ulang.
 
-A JSX node like `<List items={visibleTodos} />` is an object like `{ type: List, props: { items: visibleTodos } }`. Creating this object is very cheap, but React doesn't know whether its contents is the same as last time or not. This is why by default, React will re-render the `List` component.
+*Node* JSX seperti `<List items={visibleTodos} />` merupakan objek seperti `{ type: List, props: { items: visibleTodos } }`. Membuat objek ini sangat murah, tapi React tidak mengetahui apakah kontennya sama dengan sebelumnya atau tidak. Inilah mengapa secara bawaan, React akan me*render* ulang komponen `List`.
 
-However, if React sees the same exact JSX as during the previous render, it won't try to re-render your component. This is because JSX nodes are [immutable.](https://en.wikipedia.org/wiki/Immutable_object) A JSX node object could not have changed over time, so React knows it's safe to skip a re-render. However, for this to work, the node has to *actually be the same object*, not merely look the same in code. This is what `useMemo` does in this example.
+Namun, jika React melihat JSX yang sama persis seperti saat *render* sebelumnya, React tidak akan mencoba me*render* ulang komponen Anda. Hal ini karena node JSX [tidak dapat diubah (*immutable*).](https://en.wikipedia.org/wiki/Immutable_object) Objek *node* JSX tidak dapat diubah dari waktu ke waktu, jadi React tahu bahwa aman untuk melakukan *render* ulang. Namun, agar berfungsi, *node* harus *benar-benar menjadi objek yang sama*, bukan hanya terlihat sama dalam kode. Inilah yang dilakukan `useMemo` pada contoh ini.
 
-Manually wrapping JSX nodes into `useMemo` is not convenient. For example, you can't do this conditionally. This is usually why you would wrap components with [`memo`](/reference/react/memo) instead of wrapping JSX nodes.
+Membungkus *node* JSX secara manual ke dalam `useMemo` tidaklah mudah. Misalnya, Anda tidak dapat melakukannya secara bersyarat. Itulah mengapa biasanya Anda akan membungkus komponen dengan [`memo`](/reference/react/memo) daripada membungkus *node* JSX.
 
 </DeepDive>
 
-<Recipes titleText="The difference between skipping re-renders and always re-rendering" titleId="examples-rerendering">
+<Recipes titleText="Perbedaan antara melewati render ulang dan selalu rendering ulang" titleId="examples-rerendering">
 
-#### Skipping re-rendering with `useMemo` and `memo` {/*skipping-re-rendering-with-usememo-and-memo*/}
+#### Melewati pe-*render*-an ulang dengan `useMemo` dan `memo` {/*skipping-re-rendering-with-usememo-and-memo*/}
```

Comment:
Tidak perlu italic pada "Heading" (baca: https://github.com/reactjs/id.react.dev/wiki/Universal-Style-Guide#indonesian-specific-style-guide)

Reply:
Oops, missed that one! Jadi, penulisan yang benar itu "pe-render-an" atau "perenderan" yah?

Reply:
Penulisannya sudah benar `pe-render-an` namun untuk di "Heading" tidak perlu menggunakan italic untuk kata seperti *render*.

Jadi, untuk di Heading, cukup seperti ini:

```suggestion
#### Melewati pe-render-an ulang dengan useMemo dan memo {/*skipping-re-rendering-with-usememo-and-memo*/}
```

Reply:
Got it. Untuk code seperti `useMemo` dan `memo`, haruskah diganti juga? Namun English versionnya sudah seperti itu. (menggunakan backtick)

Reply:
@annisann Untuk nama fungsinya, kita bisa referensikan ke hal yang sudah ada https://id.react.dev/reference/react/useCallback#every-time-my-component-renders-usecallback-returns-a-different-function